Sources/AblyAssetTrackingPublisher/DefaultPublisher.swift

```diff
@@ -86,7 +75,18 @@ class DefaultPublisher: Publisher {
         self.connectionConfiguration = connectionConfiguration
         self.mapboxConfiguration = mapboxConfiguration
         self.routingProfile = routingProfile
-        self.workingQueue = DispatchQueue(label: "io.ably.asset-tracking.Publisher.DefaultPublisher", qos: .default)
+        self.workerQueue = WorkerQueue(
+            properties: PublisherWorkerQueueProperties(),
+            workingQueue: DispatchQueue(label: "io.ably.asset-tracking.Publisher", qos: .default),
+            logHandler: nil,
```

Comment:
Can we move the `self.logHandler =` line above this and pass it to the worker queue?

Reply:
Yes, done in bbd2006